backend/fms_core/tests/test_models.py

```diff
@@ -358,7 +329,7 @@ def test_negative_volume(self):
                                                   comment="ProcessSample test_negative_volume")                                                                       
                 SampleLineage.objects.create(parent=parent_sample, child=negative_volume_used, process_sample=ps)
             except ValidationError as e:
-                self.assertTrue('volume_used' in e.message_dict)
+                self.assertTrue('volume' in e.message_dict)
```

Comment:
You should use failure in test to review our code behaviour instead of "fixing" the test ;). The volume_used error is still a valid error since it point the user to the column used in the template... The validation done in ProcessSample should generate a volume_used error message here.